purchases/src/main/kotlin/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -580,6 +595,64 @@ internal class Backend(
         }
     }
 
+    fun getCustomerCenterConfig(
+        appUserID: String,
+        onSuccessHandler: (CustomerCenterConfigData) -> Unit,
+        onErrorHandler: (PurchasesError) -> Unit,
+    ) {
+        val endpoint = Endpoint.GetCustomerCenterConfig(appUserID)
+        val path = endpoint.getPath()
+        val call = object : Dispatcher.AsyncCall() {
+            override fun call(): HTTPResult {
+                return httpClient.performRequest(
+                    appConfig.baseURL,
+                    endpoint,
+                    body = null,
+                    postFieldsToSign = null,
+                    backendHelper.authenticationHeaders,
+                )
+            }
+
+            override fun onError(error: PurchasesError) {
+                synchronized(this@Backend) {
+                    customerCenterCallbacks.remove(path)
+                }?.forEach { (_, onErrorHandler) ->
+                    onErrorHandler(error)
```

Comment:
I believe most of the `onError` callbacks are already logged inside the [AsyncCall](https://github.com/RevenueCat/purchases-android/blob/main/purchases/src/main/kotlin/com/revenuecat/purchases/common/Dispatcher.kt#L47). But yeah, it's a bit mix and match currently... I think we need to refactor the `Backend` class at some point since it's getting pretty big and there is quite a bit of duplication, but not for this PR.

Reply:
Alright! Yes a quick win could be to have a (private) subclass of the `AsyncCall` that can be reused within the `Backend` class, to decrease some of the repetition. But indeed, not for this PR.